src/pages/task/Metadata.tsx

```diff
@@ -41,6 +42,12 @@ export const Metadata: React.FC<{
       <P2 data-cy="task-metadata-submitted-at">
         Submitted at: {getDateCopy(createTime)}
       </P2>
+      <P2>
+        Estimated time to start:{" "}
+        <span data-cy="task-metadata-estimated_start">
+          {getDateCopy(scheduledTime)}
```

Comment:
This should display time to start not scheduledTime to start. So it should be the time difference between `scheduledTime` and the current time